### PR TITLE
updated to use the gateway api package on pypi

### PR DESCRIPTION
**Pull request by jacoffey3**
Check it out!

### CODE REVIEW

File: run.py [hunk 1]
```diff
@@ -2,7 +2,7 @@
 import asyncio
 import time
 import argparse
-from major_tom.major_tom import MajorTom
+from mt_gateway_api.gateway_api import GatewayAPI
```
Comment (cantino) [inline run.py]: It's odd that the package is called `major-tom-gateway-api` in requirements.txt and `mt_gateway_api` when importing. Can these be the same?

Comment (jacoffey3) [reply run.py]: Yeah it was weird. Pypi's tutorial sets it up like that, and it's weird. I'd have to change the packages

**Comment by jacoffey3** [comment]
Depends on https://github.com/kubos/majortom_gateway_package/pull/4 in the gateway package

**Comment by jacoffey3** [comment]
[Update Demo Gateway to use the Gateway Package](https://trello.com/c/CGpzN4UD/15-update-demo-gateway-to-use-the-gateway-package)

**Comment by jacoffey3** [comment]
[Enable http for demo gateway](https://trello.com/c/a1KdEPT7/24-enable-http-for-demo-gateway)

File: run.py [hunk 2]
```diff
@@ -2,41 +2,35 @@
...
     '-l',
     '--loglevel',
-    help='Log level for the logger. Defaults to "debug", can be set to "info", or "error".',
-    required=False)
-optional.add_argument(
-    '-t',
-    '--telemetry',
-    help='If included, starts the telemetry beacon for the demo spacecraft automatically in the mode indicated. Can be "nominal" or "error".',
-    required=False)
+    choices=["info", "error"],
```
Comment (cantino) [inline run.py]: Probably should allow explicit selection of `debug` too?

Comment (jacoffey3) [reply run.py]: I'd rather leave it as is. It's more accurate to how the behavior is governed.

**Pull request merged**